tests/test_offload/convert/test_convert.py

```
@@ -40,16 +40,16 @@ def get_hf_dispatched_model(cuda_device, tmp_path):
 
 @pytest.mark.unit
 @requires_gpu
-def test_conversion_lifecycle(cuda_device, tmp_path):
-    model, offload_dir = get_hf_dispatched_model(cuda_device, tmp_path)
+def test_conversion_lifecycle(accel_device, tmp_path):
+    model, offload_dir = get_hf_dispatched_model(accel_device, tmp_path)
```

Ruff (0.15.9)

[warning] Unpacked variable `offload_dir` is never used

(RUF059)